admin_apps/shared_utils.py

```diff
@@ -829,261 +878,96 @@ def download_yaml(file_name: str, conn: SnowflakeConnection) -> str:
             return yaml_str
 
 
-def unpack_yaml(
-    data: Any | dict[str, Any] | list[Any]
-) -> Any | dict[str, Any] | list[str]:
+def get_sit_query_tag(
+    vendor: Optional[str] = None, action: Optional[str] = None
+) -> str:
     """
-    Recursively unpacks a YAML structure into a Python dictionary.
+    Returns SIT query tag.
+    Returns: str
     """
-    if isinstance(data, dict):
-        return {key: unpack_yaml(value) for key, value in data.items()}
-    elif isinstance(data, list):
-        return [unpack_yaml(item) for item in data]
-    else:
-        return data
 
-
-def load_yaml_file(file_paths: list[Any]) -> list[Any]:  # type: ignore
-    """
-    Loads one or more YAML files and combines them into a single list.
-    """
-    combined_yaml = []
-    for file_path in file_paths:
-        yaml_content = yaml.safe_load(file_path)
-        combined_yaml.append(unpack_yaml(yaml_content))
-    return combined_yaml
+    query_tag = {
+        "origin": "sf_sit",
+        "name": "skimantics",
```

Comment:
Is this query tag meant to be hardcoded?

Reply:
Yes, it is. I'm open to alternative implementations of course. Functions set_sit_query_tag and get_sit_query_tag are utility functions explicitly to add Solution Innovation Tag (SIT) query tags that the team uses to track solution usage and consumption. 

The official tag for SIT to track this particular effort is hard-coded in the highlighted function.